vars/chefPipeline.groovy

```diff
@@ -0,0 +1,90 @@
+def call(Closure body = null) {
+  def uuid = UUID.randomUUID().toString()
+  def pipelineParams= [:]
+
+  if (body != null) {
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = pipelineParams
+    body()
+  }
+
+  pipeline {
+    agent none
+
+    environment {
+      GITHUB_SSH_KEY = credentials('github-ssh-key')
+      TEST_KITCHEN_SSH_KEY = credentials('test-kitchen-ssh-key')
+      AWS_CREDENTIALS = credentials('aws-test-kitchen')
+      AWS_ACCESS_KEY_ID = "${env.AWS_CREDENTIALS_USR}"
+      AWS_SECRET_ACCESS_KEY = "${env.AWS_CREDENTIALS_PSW}"
+      AWS_DEFAULT_REGION = 'us-east-1'
+    }
+    stages {
+      stage('Run tests') {
+        parallel {
+          stage('Run unit tests') {
+            agent {
+              kubernetes {
+                label "leroy-unit-${uuid}"
+                containerTemplate {
+                  name 'ruby'
```

Comment:
I know the Chef CI image is based on Ruby, but if `'ruby'` still an accurate name for this? Just thinking `'chef'` might be more representative.